style.css

```diff
@@ -1,12 +1,80 @@
 * {
-  box-sizing: border-box;
-  font-family: 'Lato', sans-serif;
+    box-sizing: border-box;
+    font-family: 'Lato', sans-serif;
+}
+
+:root {
+    --color-primary: hsl(160deg, 100%, 30%);
+    --color-secondary-light: hsl(45deg, 100%, 50%);
+    --color-secondary-dark: hsl(45deg, 100%, 40%);
+    --color-black: hsl(0deg, 0%, 0%);
+    --color-grey-very-dark: hsl(0deg, 0%, 10%);
+    --color-grey-dark: hsl(0deg, 0%, 30%);
+    --color-grey-medium: hsl(0deg, 0%, 40%);
+    --color-grey-light: hsl(0deg, 0%, 60%);
+    --color-white: hsl(0deg, 0%, 100%);
 }
 
 html,
 body {
-  margin: 0;
-  padding: 0;
+    margin: 0;
+    padding: 0;
+}
+
+h1 {
+    font-size: 2rem;
+    margin-block-end: 48px;
+    margin-block-start: 32px;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+p {
+    font-size: 1rem;
+    line-height: 1.5;
+    margin-block-end: 16px;
+    margin-block-start: 16px;
+}
+
+header {
+    border-bottom: 8px solid var(--color-grey-medium);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💡 thought:** ‏It would be better to use logical properties in every case.
--> `border-block-end: 8px solid var(--color-grey-medium);`
Pixel is the right value for a border, for the font, rem will be recommanded